api/swagger.yml

```
@@ -176,28 +176,26 @@ components:
           items:
             $ref: "#/components/schemas/Repository"
 
+    MergeResultSummary:
+      description: "Deprecated: merge summary will be removed"
+      type: object
+      properties:
+        added:
+          type: integer
```

```suggestion
        added:
          type: integer
          deprecated: true
          description: Deprecated; inaccurate and will be removed.
```

(Also on all other properties, of course.)

---

@arielshaqed updated the PR with field level deprecation - note that I kept the summary as optional as I don't think we will set an empty structure if we keep it as required.